encryptme_stats/metrics.py

```diff
@@ -80,23 +80,51 @@ def _get_openvpn_stats(path="/var/run/openvpn/server-0.sock"):
     return 0
 
 
+def _get_wireguard_connections():
+    """Get number of WIREGUARD connections."""
+
+    num_wireguard = 0
+    try:
+        now_epoch = int(datetime.utcnow().timestamp())
+        interfaces = subprocess_out(["wg", "show", "interfaces"])
+        if isinstance(interfaces, list):
+            if interfaces and interfaces[-1] == '':
+                interfaces.pop()
+            for interface in interfaces:
+                peers = subprocess_out(["wg", "show", interface, "dump"])
+                if isinstance(peers, list):
+                    if peers and peers[-1] == '':
+                        peers.pop()
+                    if len(peers) < 2:
+                        continue
+                    for peer in peers[1:]:
+                        last_handshake = int(peer.split('\t')[4])
+                        # Only count if there was a handshake in the last 20 minutes
+                        if (now_epoch - last_handshake) <= (20*60): 
+                            num_wireguard += 1
+    except Exception as exc:
+        logging.debug("Error getting wireguard connections: %s", exc)
+
+    return num_wireguard
+
```

Comment:
two empty lines between functions, not 3

Reply:
Done